bugsnag-plugin-android-anr/src/test/java/com/bugsnag/android/AnrPluginTest.kt

```diff
@@ -1,37 +0,0 @@
-package com.bugsnag.android
-
-import org.junit.Assert.assertEquals
-import org.junit.Test
-
-class AnrPluginTest {
```

Comment:
Has equivalent test coverage been moved elsewhere, or does this lose unit test coverage? Ideally it'd be good to maintain or increase test coverage unless it's absolutely unavoidable.

Reply:
It's covered in the native and JVM ANR e2e tests. These tests were testing against a particular implementation, which made them very brittle to change. The e2e test checks against external results, which is resilient.